hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/MergeOperation.java

```diff
@@ -16,66 +16,85 @@
 
 package com.hazelcast.replicatedmap.impl.operation;
 
-import com.hazelcast.nio.IOUtil;
 import com.hazelcast.nio.ObjectDataInput;
 import com.hazelcast.nio.ObjectDataOutput;
 import com.hazelcast.replicatedmap.impl.ReplicatedMapService;
-import com.hazelcast.replicatedmap.impl.record.ReplicatedMapEntryView;
 import com.hazelcast.replicatedmap.impl.record.ReplicatedRecordStore;
-import com.hazelcast.replicatedmap.merge.ReplicatedMapMergePolicy;
+import com.hazelcast.spi.SplitBrainMergeEntryView;
+import com.hazelcast.spi.SplitBrainMergePolicy;
 
 import java.io.IOException;
+import java.util.LinkedList;
+import java.util.List;
 
 /**
- * Merges two replicated map entries with the given merge policy after the split-brain syndrome is recovered.
+ * Contains multiple merging entries for split-brain healing with with a {@link SplitBrainMergePolicy}.
```

Comment:
typo: "with with"

Reply:
Good catch! The `@since 3.10` was also missing.